docs/database-structure.md

```diff
@@ -0,0 +1,336 @@
+# Schema Public
+
+## Tables
+
+### audit
+
+Some exercises require reviews from other students or any other member of the school. The reviewer checks that some conditions are met by the code of the student being reviewed. This process is called an `audit` and it's kept in this table.
+
+Columns:
+- id.
+- groupId: The group that the audit refers to.
+- auditorId: The user who's making the audit.
+- attrs: The attributes of the audits keep the feedback of the auditor for each question failed.
+- grade: 0 for failed and 1 for passed.
+- createdAt: The timestamp of the time and date of creation.
+- updatedAt: The timestamp of the last update.
+- code: The code that must be used by the auditor to start the audit.
+- resultId: The result of the audit.
```

Comment:
`resultId` is filled when the group had enough audits to validate or fail a project - that means that while it's not filled, audits are still "doable" (if no grade, so not done), and then after they are not needed anymore
I feel this kind of info would be interesting in the doc (like for code) - it helps to understand how to get current audits

Reply:
This is very important. Thank you